woocommerce/woocommerce-android
# [Tablet Orders] Tracking

Closes: #10461


### Description

Tracking:
* `ORDER_FORM_TOTALS_PANEL_TOGGLED` with `flow:creation` and `expanded:true/false`
* `PAYMENTS_FLOW_ORDER_COLLECT_PAYMENT_TAPPED` with `status`, `product_count`, `has_customer_details`, `has_fees`, `has_shipping_method`, `flow`
* `flow:creation` to the payments collection events if started from the order-creating flow


```
Tracked: order_form_totals_panel_toggled, Properties: {"flow":"creation","expanded":true,"blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","store_id":"3ee2abc7-2ac9-4e18-85ae-714aeb76377a","is_debug":true,"site_url":"https:\/\/site-for-woocommerce12a3fasdf45dfs6789.mystagingwebsite.com"}

Tracked: order_form_totals_panel_toggled, Properties: {"flow":"creation","expanded":false,"blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","store_id":"3ee2abc7-2ac9-4e18-85ae-714aeb76377a","is_debug":true,"site_url":"https:\/\/site-for-woocommerce12a3fasdf45dfs6789.mystagingwebsite.com"}

Tracked: payments_flow_order_collect_payment_tapped, Properties: {"status":null,"product_types":"simple","product_count":1,"has_customer_details":false,"has_fees":false,"has_shipping_method":false,"flow":"creation","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","store_id":"3ee2abc7-2ac9-4e18-85ae-714aeb76377a","is_debug":true,"site_url":"https:\/\/site-for-woocommerce12a3fasdf45dfs6789.mystagingwebsite.com"}

Tracked: payments_flow_collect, Properties: {"payment_method":"card","order_id":2898,"flow":"creation","card_reader_type":"external","milliseconds_since_order_add_new":"74701","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","store_id":"3ee2abc7-2ac9-4e18-85ae-714aeb76377a","is_debug":true,"site_url":"https:\/\/site-for-woocommerce12a3fasdf45dfs6789.mystagingwebsite.com"}
```

### Testing instructions

* Use logcat with `track` filter
* Orders -> Start Order Creation -> Add Products -> collect payment
* Notice tracking
